Makefile

```diff
@@ -960,12 +960,19 @@ shell: $(GOBUILDER)
 	$(QUIET)DOCKER_GO_ARGS=-t ; $(DOCKER_GO) bash $(GOTREE) $(GOMODULE)
 
 #
-# Utility targets in support of our Dockerized build infrastrucutre
+# Linuxkit
 #
+.PHONY: linuxkit
+linuxkit: $(LINUXKIT)
+
+LINUXKIT_SOURCE=https://github.com/linuxkit/linuxkit
+
+$(LINUXKIT_BIN_DIR):
+	mkdir -p $@
 
-# check to make sure linuxkit version is correct
-# if it does not exist, version is incorrect, or it cannot report `version --short`, download it
-$(LINUXKIT): FORCE
+$(LINUXKIT): $(LINUXKIT_BIN_DIR)
+#   check to make sure linuxkit version is correct
+#   if it does not exist, version is incorrect, or it cannot report `version --short`, download it
 	$(eval ACTUAL_LINUXKIT_VERSION := $(strip $(shell $@ version --short 2>/dev/null || echo "unknown")))
```

Comment:
I think we can even remove these checks and just keep `curl` part but it won't hurt. @deitch what do you think?